src/domain/occurrences/OccurrenceShow.tsx

```diff
@@ -130,7 +129,7 @@ const OccurrenceShow = () => {
           label="occurrences.fields.capacity.label"
         />
         <FunctionField
-          render={(occurrence: Partial<OccurrenceNode>) =>
+          render={(occurrence: RaRecord) =>
```

Comment:
```suggestion
          render={(occurrence: OccurrenceNode) =>
```
How about using `OccurrenceNode` type here? I tried to look into the code to see how it determines the input data type here but I couldn't find it with a quick glance. UseRecordContext hook is used somewhere? Just looking at the types that do have `freeSpotNotificationSubscriptions` in it the codebase had `ChildNode` and `OccurrenceNode` and as this is `OccurrenceShow.tsx` then the latter seems like the more fitting choice here.

Reply:
We had a discussion and there was a problem with "TS2589: Type instantiation is excessively deep and possibly infinite" coming up using VSCode with `Partial<OccurrenceNode>` or `OccurrenceNode`. While making the types worse is not ideal, I don't see this is a complete blocker here. If using `RaRecord` here makes VSCode's type checking work ok, this is ok now. If we'd be removing more and more types from kukkuu-admin's source code, then I think it'd be good to try to find the cause of the VSCode type checking problems and fix them. Maybe something to do with newer typescript version or VSCode's typescript checking plugin/language server?